pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -20,6 +20,11 @@ import (
 )
 
 func validateInput(c utils.CommandLine, pluginFolder string) error {
+	if c.Args().Len() > 1 {
+		logger.Info("Install only supports one local argument\n")
+		showIgnoredArguments(c.Args().Slice()[1:])
+	}
```

Comment:
Also, the length to check should be 2, not 1 (it's possible to specify the version, apart from the name). This has the caveat that if you specify `... install foo --pluginsDir=bar`, the number of arguments is correct but it will fail anyway when trying to use the flag as a version.